pyinstaller-gui.py

```diff
@@ -76,7 +76,11 @@ def killapp(self, event):
         sys.exit(0)
 
     def makePackage(self, event):
-        commands = ['python3', 'pyinstaller.py']
+        if sys.platform.startswith('win'):
+            pythoncmd = ['py', '-3']
+        else:
+            pythoncmd = ['python3']
+        commands = pythoncmd + ['pyinstaller.py']
```

Comment:
IMHO we can set `commands` directly in the if-else-clause. Adding an additional variable `pycommand` does not make much sense as the whole statement is simple.